internal/test/daemon/daemon.go

```diff
@@ -630,7 +630,9 @@ func (d *Daemon) getClientConfig() (*clientConfig, error) {
 		return nil, err
 	}
 	transport.DisableKeepAlives = true
-
+	if proto == "unix" {
+		addr = filepath.Base(addr)
```

Comment:
can be just `addr = "dummy"`?

Reply:
yes, I was in doubt; was thinking if having the ID of the daemon in there could be useful in other places for debugging.

Happy to change if you prefer "dummy", "localhost",  "moby" or "whatever"

Reply:
`addr="<path>"` maybe?